app/extensions/brave/locales/en-US/extensions.properties

```diff
@@ -3,3 +3,29 @@ extensions=Extensions
 extensionIdLabel=ID:
 extensionPathLabel=Path:
 extensionPermissionsLabel=Permissions:
+notifyNewExtensionsAdded=Notifiy me when new extensions are added
+extensionsTabFooterInfo=Looking for a favorite extension? Let us know through our
+latest=Latest
+notInstalled=Not Installed
+integrated=Integrated
+
+l10nBitwarden=bitwarden - Free Password Manager
+l10nBitwardenDesc=bitwarden is a secure and free password manager for all of your devices.
+l10nDashlane=Dashlane Secure Password Manager
+l10nDashlaneDesc=Never forget another password. Dashlane password manager - the most secure password storage and fast login on all of your devices
+l10nEnpass=Enpass Password Manager
+l10nEnpassDesc=Enpass Password Manager extension for Brave
+l10nLastpass= LastPass: Free Password Manager
+l10nLastpassDesc=LastPass, an award-winning password manager, saves your passwords and gives you secure access from every computer and mobile device.
+l10nOnepassword= 1Password: Password Manager and Secure Wallet
+l10nOnepasswordDesc=1Password extension for Brave.
+l10nPdfjs= PDF Viewer
+l10nPdfjsDesc=Uses HTML5 to display PDF files directly in the browser.
+l10nPocket= Save to Pocket
+l10nPocketDesc=Pocket Extension for Brave - The best way to save articles, videos and more.
+l10nSync= Brave Sync
+l10nSyncDesc=
+l10nWebtorrent= Torrent Viewer
+l10nWebtorrentDesc=
+l10nVimium= Vimium
```

Comment:
We'll be shipping Honey soon (perhaps in 0.14.2). Can we include it?
https://chrome.google.com/webstore/detail/honey/bmnlcjabgnpnenekpadlanbbkooimhnj